hazelcast/src/test/java/com/hazelcast/config/DiscoveryStrategyConfigTest.java

```diff
@@ -84,6 +84,20 @@ public void test_DiscoveryStrategyFactory_constructor_factory_properties() {
         assertEquals(properties, discoveryStrategyConfig.getProperties());
     }
 
+    @Test
+    public void test_DiscoveryStrategyFactory_properties_add_remove() {
+        DiscoveryStrategyFactory discoveryStrategyFactory = new TestDiscoveryStrategyFactory();
+        DiscoveryStrategyConfig discoveryStrategyConfig = new DiscoveryStrategyConfig(discoveryStrategyFactory);
+        assertEquals(0, discoveryStrategyConfig.getProperties().size());
```

Comment:
I would prefer `assertTrue(discoveryStrategyConfig.getProperties().isEmpty())` but yours is ok too.